src/core/src/basics/song.cpp

```diff
@@ -76,7 +76,9 @@ Song::Song( const QString& name, const QString& author, float bpm, float volume
 	, __is_loop_enabled( false )
 	, __humanize_time_value( 0.0 )
 	, __humanize_velocity_value( 0.0 )
-	, __swing_factor( 0.0 )
+	  // Colors: 0 - white noise; 1 - pink noise
```

Comment:
Yes, I also like the idea of using an *enum class* to store the color of the noise instead of just an integer value. But the color needs to be saved to the .h2song file. Therefore I would vote against using such a class since this would require some additional overhead in converting it both in the MixerLine and the LocalFileMng/Song instance. 

What do you think?